app.py

```diff
@@ -189,7 +192,7 @@ def update_graph_content(selected_model):
             dmc.Center(dmc.Text("PPD = 0.06")),
             dmc.Center(dmc.Text("Category = |")),
         ]
-    if selected_model == 'PMV - ASHRAE 55':
```

Comment:
there should not be an if here, the output should be updated as a funtion of the model output and it should not be hardcoded.